pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2420,6 +2421,10 @@ public boolean isAllowAutoTopicCreation(final String topic) {
     }
 
     public boolean isAllowAutoTopicCreation(final TopicName topicName) {
+        //System topic can always be created automatically
+        if (EventsTopicNames.NAMESPACE_EVENTS_LOCAL_NAME.equals(topicName.getLocalName())) {
```

Comment:
It should be created automatically when enabling the system topic?

Reply:
Sorry, I overlooked this